nodejs/week2/warm_up/index.js

```diff
@@ -0,0 +1,37 @@
+//Lets use express routes to create a small calculator:
+
+const express = require("express");
+const app = express();
+const port = 3000;
+
+app.get('/numbers/add', (req, res) => {
+    let sum = Number(req.query["first"]) + Number(req.query["second"]);
```

Comment:
I think` req.query["first"]` is overcomplicated.
You can use just `req.query.first` and get the same value.

Reply:
Implemented the given feedback